plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestParquetPageSkipping.java

```diff
@@ -159,6 +170,57 @@ public void testPageSkipping(String sortByColumn, String sortByColumnType, Objec
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testFilteringWithColumnIndex()
```

Comment:
can we have such tests in parquet module (maybe UT) instead to improve locality?

Reply:
There are many UTs about column index filtering in `io.trino.parquet.reader.TestColumnReader`
Verifying that things work as expected in actual queries is important.